django_mailbox/models.py

```diff
@@ -450,6 +457,11 @@ class Message(models.Model):
         null=True,
     )
 
+    eml = models.FileField(
```

Comment:
Since we have to support earlier versions of Django (1.4.x+), could I convince you into setting the `upload_to` keyword argument, here, to something like 'messages'.